Backend/routes/delete-req.js

```diff
@@ -0,0 +1,18 @@
+const ans = require('../db/api_call');
+// let readKey = '2' // will make it dynamic
+// let action = 'remove/' + readKey
+
+var deleteReq = async (req, res) => {
+    let temp = ""

```

Comment:
you dont have to initialize it as an empty string first you can just set this variable below